mitodl/ol-infrastructure
# set failed pipeline garbage collection grace period to 5 minutes

# What are the relevant tickets?
Related to https://github.com/mitodl/ocw-studio/issues/1966

# Description (What does it do?)
The new `mass-build-sites` pipeline definition generated by `ocw-studio` and used in Concourse runs batches of parallel OCW site builds using `across`, generating very large amounts of container usage across all OCW workers. If one of these builds fails or is aborted, the current strategy of having them garbage collected every 2 hours means 500+ containers hanging around that long. This PR sets that to 5 minutes, so in the event of a failure Concourse becomes usable more quickly

# How can this be tested?
Deploy the setting, trigger a build, abort it, then wait 5 minutes and see if container counts drop
